### DISCUSSION TITLE
Stable Cascade

### DISCUSSION THREAD

**Discussion by logikstate** [Acly/krita-ai-diffusion#411] [Ideas]
Anybody seen this? Brand new model from Stable Diffusion... looks like it has Comfy UI nodes... Its also apparently good at text.

https://www.youtube.com/watch?v=Ybu6qTbEsew

**Comment by logikstate** [comment]
This video demonstrates text/logo generation on a hugging face repo https://youtu.be/-ICRqD3pYXw

**Comment by Acly** [comment]
Yes, waiting for official ComfyUI support to see how it fits in.

Keep in mind it took some months for SDXL to have all the finetunes, control, inpaint, etc. to be useful. So the most exciting part about Cascade is IMO that it's (supposedly) easier to train in these areas.
